open_seq2seq/parts/centaur/conv_block.py

```diff
@@ -0,0 +1,78 @@
+import tensorflow as tf
```

Comment:
Can we move this to parts/cnns/conv_blocks.py? Maybe call it casual_conv?

Reply:
We decided not to do it.